src/backend/api/meals.js

```diff
@@ -1,15 +1,83 @@
-const express = require("express");
-const router = express.Router();
-const knex = require("../database");
+ const express = require("express");
+ const router = express.Router();
+ const knex = require("../database");
 
-router.get("/", async (request, response) => {
+ router.get("/", async (request, response) => {
+   try {
+    // knex syntax for selecting things. Look up the documentation for knex for further info
+     const meals = await knex("meal").select("*");
+     if(meals.length === 0){
+      response.status(404).send("No meals available !")
+     }
+      response.json(meals);
```

Comment:
I suggest using else extension to if condition here, because `reponse.json(meals); ` sends the empty meals array, when there are no meals.

Reply:
yes true !